libs/dh/wholesale/feature-calculation-steps/src/lib/time-series-points/dh-wholesale-time-series-points.component.scss

```diff
@@ -19,8 +19,18 @@
   width: 100%;
 }
 
+watt-card {
```

Comment:
Question: could we do some context-specific styling with ex. `:host-context('something something')` for the card, so we don't have to do this in multiple places?

Reply:
Oh cool, wasn't aware of `:host-context`, good idea! Will check it out 🙂